src/tests/ftest/soak/soak.yaml

```diff
@@ -1,146 +1,160 @@
-        daos_servers:
-            - boro-A
-        # either compute or partition can be used, not both
-        #compute:
-        #    - boro-A
-        partition: tp
-        gateway: []
-timeout: 1000
+    test_servers:
+        - boro-A
+        - boro-B
+        - boro-C
+        - boro-D
+    test_clients:
+        - boro-E
+        - boro-F
+        - boro-G
+        - boro-H
+        - boro-I
+        - boro-J
+        - boro-K
+        - boro-L
+timeout: 7200
 server_config:
     name: daos_server
-
+#slurm partition names
+partition:
+        client: daos_client
+        server: daos_server
+# pool_params - attributes of the pools to create; Currently only create one
+# pool
 pool1:
     createmode:
-        mode_RW:
-             mode: 146
+        mode_RW: 146
     createset:
         setname: daos_server
     createsize:
-        size: 10000000000
+        size: 100000000000
     createsvc:
-        svcn: 1
-
+        svcn: 3
+# test_params - Defines the type of test to run and how long it runs
+#               It also defines how many pools and jobs to create
+#               name:                The name of the Avocado testcase
+#               test_timeout:        The overall timeout in secs; soak12 will be
+#                                    changed to 43200 once it is stable
+#               test_iteration:      values 1 or -1; -1 is used to cause the
+#                                    IOR -T x to end cmd.  i = 100000000
+#                                    (does not seem to work)
+#               nodesperjob:         slurm -N param; -1 indicates all nodes
+#                                    in -partition
+#               poollist:            defines pools to create for jobs
+#               joblist:             defines workload per slurm scripts
+# smoke testcase params
+smoke:
+    name: smoke
+    test_timeout: 1800
+    test_iteration: 1
+    nodesperjob: -1
+    pool: pool1
+    joblist:
+      - jobsmoke
+# 1 hours testcase params
+soak1:
+    name: soak1
+    test_timeout: 3600
+    test_iteration: 5
+    nodesperjob: 1
+    pool: pool1
+    joblist:
+      - job1
+      - job16
+      - job24
+# job_params  - defines the job specifications for each cmdline listed
+#               name:                The name of the Avocado testcase
+#               time:                The IOR cmdline timeout; passed to slurm
+#               tasks:               slurm --ntaskspernode param
+#               blocksize:           IOR blocksize; need to consider poolsize
+#                                    and total ntasks (nodes * ntaskspernode)
+#                                    This param will be determined in the
+#                                    test later.
+#               jobspec:             list of cmdline specifications to run
+# smoke job params; all nodes on client partition are used and one taskpernode
+jobsmoke:
+    name: smoke
+    time: 10
+    tasks: 1
+    blocksize: '64M'
+    jobspec:
+      - ior3
```

Comment:
Is there a reason why we wouldn't want these to be more descriptive, i.e.
`ior1 `->`ior_daos_4k_large`
`ior2 `->`ior_doas_128k_small`
`ior3 `->`ior_daos_1m_large`
`ior4 `->`ior_mpiio_4k_tiny`
`ior5 `->`ior_mpiio_128k_large`
`ior6 `->`ior_mpiio_1m_large`